packages/components/TeamsList/TeamsList.js

```diff
@@ -31,16 +31,30 @@ const Option = styled.div`
 const StyledAvatar = styled(Avatar)`
   margin-right: 16px;
 `;
+const sortTeams = (a, b) => {
+  if (a.title.toLowerCase() === TEAM_TYPE.DEFAULT) return 1;
+  if (b.title.toLowerCase() === TEAM_TYPE.DEFAULT) return -1;
 
+  return sortByName(a.title, b.title);
+};
+const isTeamEnable = activeTeamId => team => {
+  if (!team?.id) return false;
+  // Don't show the active team
+  const isNonActiveTeam = team?.id !== activeTeamId;
+  // Always show the must and pinned teams
+  const isMustTeams =
```

Comment:
what does 'isMustTeams' mean?

Reply:
@cu4xxi isMust(be shown)Teams: you can't hide the domain or personal teams.